core/services/scheduler_test.go

```diff
@@ -79,6 +80,56 @@ func TestRecurring_AddJob(t *testing.T) {
 	runManager.AssertExpectations(t)
 }
 
+func TestRecurring_AddJob_PastEnd(t *testing.T) {
+	store, cleanup := cltest.NewStore(t)
+	defer cleanup()
+
+	runManager := new(mocks.RunManager)
```

Comment:
In favor of the "do things one way" principle, I'd be inclined to push back on using `new` anywhere in the codebase.  Literal syntax can cover all cases that `new` handles, while `new` handles no cases that literal syntax can't.